usr/sbin/registercloudguest

```diff
@@ -451,425 +453,440 @@ argparse.add_argument(
 )
 
 
-args = argparse.parse_args()
+def main(args):
+    global registration_returncode
```

Comment:
while this is hard to review, I'm just asking. The change here was adding a main() method and all code was moved into it right ? or did you also change code paths that you want us to review ?

Thanks

Reply:
correct, introduction to `main(args)` method